Workbooks/Usage/categoryResources.json

```diff
@@ -1,3 +1,4 @@
 {
-    "en-us": {"name":"Usage", "description": "", "order": 400}
+    "en-us": {"name":"Usage", "description": "", "order": 400},
+    "es.es.-es": {"name":"Uso", "description": "", "order": 400}
```

Comment:
why is this `es.es.-es` ?  shouldn't it be like `es-mx` or `es-es` for mexico vs spain?  what are the exact values used by the portal?

Reply:
thats what the portal was showing for language, but changed it to es-es